.template/addons/github/.github/workflows/test.yml.tt

```diff
@@ -0,0 +1,76 @@
+name: Test
+
+on: push
+
+env:
+  DOCKER_REGISTRY: ${{ secrets.DOCKER_REGISTRY }}
+  DOCKER_USERNAME: ${{ github.repository_owner }}
+  DOCKER_TOKEN: ${{ secrets.DOCKER_TOKEN }}
+  COMPOSE_FILE: docker-compose.test.yml
+  DOCKER_IMAGE: ${{ github.repository }}
+
+jobs:
+  build:
+    name: Build docker image
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2.3.2
+
+      - uses: nimblehq/branch-tag-action@v1.1
+
+      - name: Docker login
+        run: echo "$DOCKER_TOKEN" | docker login $DOCKER_REGISTRY --username=$DOCKER_USERNAME --password-stdin
+
+      - name: Docker pull
+        if: env.BRANCH_TAG != 'latest' && env.BRANCH_TAG != 'development'
+        run: docker-compose pull test || true
```

Comment:
@hoangmirs I have a question, what is the difference between
- docker-compose pull
- docker-compose pull test
- docker-compose pull test || true

Reply:
1. It will pull all the sevices that we defined on docker-compose file
2. It will pull the test service that we defined on docker-compose file
3. Same as 2 but it won't stop the workflow if there is an error when pulling